aspnetcore/mobile/native-mobile-backend.md

```diff
@@ -46,11 +47,49 @@ public static string RestUrl = "http://192.168.1.207:5000/api/todoitems/{0}";
 
 ## Creating the ASP.NET Core Project
 
-Create a new ASP.NET Core Web Application in Visual Studio. Choose the Web API template and No Authentication. Name the project *ToDoApi*.
+# [Visual Studio](#tab/visual-studio)
```

Comment:
Delete all duplication from https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api
Send them to https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api for the API project.

This topic should only have unique content.